packages/eas-cli/src/credentials/ios/actions/new/SetupAdhocProvisioningProfile.ts

```diff
@@ -22,38 +21,34 @@ import { chooseDevices } from './DeviceUtils';
 import { doUDIDsMatch, isDevPortalAdhocProfileValid } from './ProvisioningProfileUtils';
 import { SetupDistributionCertificate } from './SetupDistributionCertificate';
 
-export class SetupAdhocProvisioningProfile implements Action {
-  private _iosAppBuildCredentials?: IosAppBuildCredentialsFragment;
-
+export class SetupAdhocProvisioningProfile {
   constructor(private app: AppLookupParams) {}
 
-  public get iosAppBuildCredentials(): IosAppBuildCredentialsFragment {
-    assert(
-      this._iosAppBuildCredentials,
-      'iosAppBuildCredentials can be accessed only after calling .runAsync()'
-    );
-    return this._iosAppBuildCredentials;
-  }
-
-  async runAsync(manager: CredentialsManager, ctx: Context): Promise<void> {
+  async runAsync(
+    manager: CredentialsManager,
```

Comment:
If the manager needs to passed to other classes keep `implements Action<IosAppBuildCredentialsFragment>` in the declaration if it's not necessary then remove it